api/ruxos_posix_api/src/imp/net.rs

```diff
@@ -21,6 +21,26 @@ use ruxnet::{TcpSocket, UdpSocket};
 use crate::ctypes;
 use crate::utils::char_ptr_to_str;
 
+const SOL_SOCKET: i32 = 1;
```

Comment:
Remove these definations and use `ctyes::SO_BROADCAST` to make sure they are consistent, if there is no special reason.